addon/templates/components/frost-navigation.hbs

```diff
@@ -17,4 +16,7 @@
   <!-- User Menu -->
   {{component menu}}
 </div>
-{{liquid-modal}}
+{{to-elsewhere
```

Comment:
I think you want to make your `elsewhere` conditional here so that we only send the component when the navigation is active